android/src/main/java/com/matejdr/admanager/BannerAdView.java

```diff
@@ -118,7 +133,114 @@ public void onAdClosed() {
             }
 
         });
+
+        updateLayout();
+
         this.addView(this.adView);
+
+        updateLayout();
```

Comment:
can the first `updateLayout()` be omitted?

Reply:
This is no longer needed as we delay the layout until we know the ad size (only after it was loaded).